apps/server/src/modules/board/controller/api-test/content-element-delete.api.spec.ts

```diff
@@ -134,4 +141,60 @@ describe(`content element delete (api)`, () => {
 			expect(response.status).toEqual(403);
 		});
 	});
+
+	describe('for drawing element', () => {
+		const drawingSetup = async () => {
+			await cleanupCollections(em);
+			const teacher = userFactory.asTeacher().build();
+			const student = userFactory.asStudent().build();
+			const course = courseFactory.build({ teachers: [teacher], students: [student] });
+			await em.persistAndFlush([teacher, student, course]);
+
+			const columnBoardNode = columnBoardNodeFactory.buildWithId({
+				context: { id: course.id, type: BoardExternalReferenceType.Course },
+			});
+			const columnNode = columnNodeFactory.buildWithId({ parent: columnBoardNode });
+			const cardNode = cardNodeFactory.buildWithId({ parent: columnNode });
+			const element = drawingElementNodeFactory.buildWithId({ parent: cardNode });
+
+			jest.spyOn(filesStorageClientAdapterService, 'deleteFilesOfParent').mockResolvedValueOnce([]);
```

Comment:
avoid jest.spyOn usage, on the other api test you don't have it

instead, with DeepMock you should be able to have something like filesStorageClientAdapterService.deleteFilesOfParent.mockResolvedValueOnce([]); 
typically, we don't need to mock in api tests, but here it's needed. Look in the uc.spec for example how we mock

Reply:
fixed